packages/base-zone/src/prepare-revocable.js

```diff
@@ -153,3 +185,83 @@ export const prepareRevocableMakerKit = (
   });
 };
 harden(prepareRevocableMakerKit);
+
+/**
+ * Create an exo class kit used to guard operations and cancel them.
```

Comment:
I'm struggling to understand why use this in `assetReserveKit.js`

If it's worth doing, I **suggest** that it should have unit tests.